### PR TITLE
Readme module import update

### PR DESCRIPTION
**Pull request by hrishiballal**
Added Instructions to run DSS locally

### CODE REVIEW

File: introduction_to_repository.md [hunk 1]
```diff
@@ -9,7 +9,12 @@ This document aims to provide a introduction to the repository and it's structur
 
 ### Monitoring toolset
 
-The `monitoring` directory contains a set of folders containing different test suites to test different capabilities of the DSS during development and production use.
+The `monitoring` directory contains a set of folders containing different test suites to test different capabilities of the DSS during development and production use. 
+
+### Running tools locally
+
+- When running tools in the monitoring toolset for local debugging outside of Docker, the [monitoring](https://github.com/interuss/dss/tree/master/monitoring) top-level folder must be added to your `PYTHONPATH`.
```
Comment (BenjaminPelletier) [inline introduction_to_repository.md:16]: More precisely, the `monitoring` folder must [be accessible from the user's PYTHONPATH](https://github.com/interuss/dss/blob/8c9acd0c6659bde2c1884361776f501a17f3dd0d/monitoring/tracer/Dockerfile#L13).  How about:

```markdown
When running tools in the monitoring toolset for local debugging outside of Docker, the [monitoring](https://github.com/interuss/dss/tree/master/monitoring) folder must be accessible from one of the entries in your `PYTHONPATH`.  To accomplish this, add the root repo folder to your `PYTHONPATH`.
```

File: introduction_to_repository.md [hunk 2]
```diff
@@ -9,7 +9,12 @@ This document aims to provide a introduction to the repository and it's structur
...
+- To run a DSS instance locally, you will need to run the `run_locally.sh` in the [/build/dev/](https://github.com/interuss/dss/blob/master/build/dev/run_locally.sh) directory. This will setup a local instance of the DSS and the Auth server at `http://localhost:8085`.
```
Comment (BenjaminPelletier) [inline introduction_to_repository.md:17]: Let's refer the reader to where there is more information:

```markdown
To run a DSS instance locally, see [this documentation](build/dev/stand_alone_instance.md).
```

File: introduction_to_repository.md (hunk 1 above)
Comment (hrishiballal) [reply introduction_to_repository.md:16]: Thanks fixed now!

File: introduction_to_repository.md (hunk 2 above)
Comment (hrishiballal) [reply introduction_to_repository.md:17]: Fixed this now, thanks!

**Review by BenjaminPelletier — APPROVED** [review]

**Comment by BenjaminPelletier** [comment]
Just to be clear on status: this PR is currently awaiting resolution of conflicts with master branch

**Pull request merged**